File: app2/src/main/java/com/ssz/framejava/module/home/HomePresenter.java
package com.ssz.framejava.module.home;


import com.ssz.framejava.base.func.ISuccessListener;

import io.reactivex.disposables.Disposable;

/**
 * @author : zsp
 * time : 2019 10 2019/10/10 13:18
 */
public class HomePresenter implements IHomeContract.IPresenter {

    private IHomeContract.IView mView;

    @Override
    public void attach(IHomeContract.IView IView) {
        this.mView = IView;
    }

    @Override
    public void detach() {
        this.mView = null;
    }

    @Override
    public Disposable login() {
        Disposable d = null;

        return d;
    }

    @Override
    public Disposable login2(ISuccessListener<Boolean> listener) {
        listener.result(false);
        return null;
    }
}

File: app/src/main/java/com/ssz/studydemo/view/HomePresenter.kt
package com.ssz.studydemo.view

import com.ssz.studydemo.view.home.IHomeContract

/**
 * @author : zsp
 * time : 2019 09 2019/9/17 15:11
 */
class HomePresenter(val view : IHomeContract.IHomeView) : IHomeContract.IHomePresenter {

    init {
        view.mPresenter = this
    }

    override fun attach() {

    }

    override fun detach() {

    }
}